src/effects.ts

```diff
@@ -0,0 +1,203 @@
+import type { AScene } from "aframe";
+import {
+  BlendFunction,
+  BloomEffect,
+  CopyPass,
+  EffectComposer,
+  EffectPass,
+  LambdaPass,
+  RenderPass,
+  SMAAEffect,
+  ToneMappingEffect,
+  ToneMappingMode
+} from "postprocessing";
+import { Camera, HalfFloatType, Mesh, OrthographicCamera, Scene, WebGLRenderer } from "three";
+import { Layers } from "./camera-layers";
+import { createImageMesh } from "./utils/create-image-mesh";
+import Store from "./storage/store";
+import qsTruthy from "./utils/qs_truthy";
+import ImageMessage from "./react-components/image-message";
+
+export enum AAModes {
+  NONE = "NONE",
+  SMAA = "SMAA",
+  MSAA_2X = "MASS_2X",
+  MSAA_4X = "MASS_4X",
+  MSAA_8X = "MASS_8X"
+}
```

Comment:
Just wondering whether we really need five aa options because it may add complexity in the preferences. For example aren't just two options (ON/OFF) good? No big preference tho...

Reply:
I think we want at least off, SMAA, and some form of MSAA. I mostly included all of these so that people can test different options for visual differences and perf on different devices.

Reply:
I did notice the constants for MSAA all have a typo though.. will fix